app/views/home/donate.html.erb

```diff
@@ -1,12 +1,13 @@
-﻿<h2 class="heading"><%= ts('Donations') %></h2>
-<h3 class="heading"><%= ts('There are two main ways to support the AO3 - donating your time or money.') %></h3>
+﻿<h2 class="heading"><%= t(".general.title") %></h2>
+<h3 class="heading"><%= t(".general.text") %></h3>
 
 <div class="userstuff">
-  <h3><%= ts('Donating Your Time') %></h3>
-  <p><%= ts("The <a href=\"http://www.transformativeworks.org\">Organization for Transformative Works</a> is the parent organization of the Archive of Our Own. We are often looking for volunteers to contribute to <a href=\"http://www.transformativeworks.org/our-projects\">our projects</a>. If you're interested in volunteering specifically to help the Archive of Our Own, the roles to look out for include tag wranglers, testers, coders, Support staff and Abuse staff. We encourage you to browse through the <a href=\"http://www.transformativeworks.org/volunteer\">available listings</a> and apply for any roles which match your qualifications and interests.").html_safe %></p>
+  <h3><%= t(".time.title") %></h3>
+  <p><%= t(".time.text_html",
+           otw_url: link_to(t(".time.otw_url_text"), t(".time.otw_url")),
```

Comment:
I think we could also rename the interpolated variables with a `_link` suffix according to the i18n guidelines, to distinguish them from the actual url.

Reply:
Yeah, sorry, I meant do it exactly like the emails, with the URL hard coded and the link text variable:

```
# I went with details_html rather than text_html to avoid any confusion about whether this is plain text or HTML
<%= t(".time.details_html",
           otw_link: link_to(t(".time.otw"), http://www.transformativeworks.org),
           projects_link: link_to(t(".time.projects"), http://www.transformativeworks.org/our-projects),
           volunteer_link: link_to(t(".time.volunteer"), http://www.transformativeworks.org/volunteer)) %></p>
```

That's the pattern we've established with our Translation team, so we should stick to it.